deb.mk

```diff
@@ -140,16 +140,16 @@ endif
 ifneq (,$(findstring false,$(IS_RELEASE)))
 	sed -i.bak -e "s/^/`date +%s`:/" /tmp/build/var/lib/dokku/STABLE_VERSION && rm /tmp/build/var/lib/dokku/STABLE_VERSION.bak
 endif
-	rm /tmp/build/DEBIAN/lintian-overrides
-	mv debian/lintian-overrides /tmp/build/usr/share/lintian/overrides/dokku
+	rm -f /tmp/build/DEBIAN/lintian-overrides
+	cp debian/lintian-overrides /tmp/build/usr/share/lintian/overrides/dokku
 ifdef DOKKU_GIT_REV
 	echo "$(DOKKU_GIT_REV)" > /tmp/build/var/lib/dokku/GIT_REV
 else
 	git rev-parse HEAD > /tmp/build/var/lib/dokku/GIT_REV
 endif
-	sed -i "s/^Version: .*/Version: `cat /tmp/build/var/lib/dokku/STABLE_VERSION`/g" /tmp/build/DEBIAN/control
-	dpkg-deb --build /tmp/build "/tmp/dokku_`cat /tmp/build/var/lib/dokku/STABLE_VERSION`_$(DOKKU_ARCHITECTURE).deb"
-	lintian "/tmp/dokku_`cat /tmp/build/var/lib/dokku/STABLE_VERSION`_$(DOKKU_ARCHITECTURE).deb"
+	sed -i.bak "s/^Version: .*/Version: `cat /tmp/build/var/lib/dokku/STABLE_VERSION`/g" /tmp/build/DEBIAN/control && rm /tmp/build/DEBIAN/control.bak
```

Comment:
That's not necessarily a reason to leave it as is though. In any case, if you're comfortable with maintaining the pattern, then that's cool with me.